pkg/common/did/creator_test.go

```diff
@@ -26,15 +26,30 @@ func getMockDID() *d.Doc {
 	}
 }
 
-func TestLocalDIDCreator_CreateDID(t *testing.T) {
-	creator := NewLocalDIDCreator(&mockProvider{})
-	require.NotNil(t, creator)
+func TestLocalDIDCreator(t *testing.T) {
 
-	did, err := creator.CreateDID()
-	require.NotNil(t, did)
-	require.NoError(t, err)
+	t.Run("create new local DID ", func(t *testing.T) {
+		creator := NewLocalDIDCreator(&mockProvider{})
+		require.NotNil(t, creator)
+
+		did, err := creator.CreateDID()
+		require.NotNil(t, did)
+		require.NoError(t, err)
+
+		require.Equal(t, did, getMockDID())
+	})
+
+	t.Run("get DID ", func(t *testing.T) {
+		creator := NewLocalDIDCreator(&mockProvider{})
+		require.NotNil(t, creator)
+
+		did, err := creator.GetDID(getMockDID().ID)
+		require.NotNil(t, did)
+		require.NoError(t, err)
+
+		require.Equal(t, did, getMockDID())
+	})
 
```

Comment:
remove this empty line since below line is deleted... this is causing linter check error

Reply:
done